integration-tests/helpers/embrace_mock_api.ts

```diff
@@ -0,0 +1,37 @@
+import {ParsedLogPayload, ParsedSpanPayload} from "../typings/embrace";
+import axios from "axios";
+import {parseSpanPayload} from "./span";
+import {parseLogPayload} from "./log";
+
+const EMB_URL = "https://mock-api.emb-eng.com/namespace/SgNw5/stored";
```

Comment:
namespace needs to be dynamic here

Reply:
Yes the other day you told me that instead of env you are going to do something with params or something like that

Reply:
in integration-tests/wdio.conf.ts there is `parseCLIOptions` that you can add more options to